doc/languages-frameworks/idris.section.md

```diff
@@ -11,10 +11,21 @@ $ # On non-NixOS
 $ nix-env -i nixpkgs.idris
 ```
 
-This however only provides the `prelude` and `base` libraries. To install additional libraries:
+This however only provides the `prelude` and `base` libraries. To install idris with additional libraries, you can use the `idrisPackages.with-packages` function, e.g. in an overlay in `~/.config/nixpkgs/overlays/my-idris.nix`:
 
+```nix
+self: super: {
+  myIdris = with self.idrisPackages; with-packages [ contrib pruviloj ];
+}
 ```
-$ nix-env -iE 'pkgs: pkgs.idrisPackages.with-packages (with pkgs.idrisPackages; [ contrib pruviloj ])'
+
+And then:
+
+```
+$ # On NixOS
+$ nix-env -iA nixos.idris
+$ # On non-NixOS
+$ nix-env -iA nixpkgs.idris
```

Comment:
should this use `my-idris`? as in `nix-env -iA nixpkgs.my-idris`?

Reply:
Ah wow how did I miss that, yes, but with `myIdris` because that's the attribute name